docs/sources/flow/reference/components/otelcol.connector.spanmetrics.md

```diff
@@ -184,6 +199,348 @@ The following fields are exported and can be referenced by other components:
+                           "key": "service.name",
+                           "value": { "stringValue": "TestSvcName" }
+                         },
+                         {
+                           "key": "span.name",
+                           "value": { "stringValue": "TestSpan" }
+                         },
+                         {
+                           "key": "span.kind",
+                           "value": { "stringValue": "SPAN_KIND_UNSPECIFIED" }
+                         },
+                         {
+                           "key": "status.code",
+                           "value": { "stringValue": "STATUS_CODE_UNSET" }
+                         }
+                       ],
+                       "startTimeUnixNano": "1702582936761872000",
+                       "timeUnixNano": "1702582936761872012",
+                       "asInt": "1"
+                     }
+                   ],
+                   "aggregationTemporality": 2,
+                   "isMonotonic": true
+                 }
+               }
+             ]
+           }
+         ]
+       },
+       {
+         "resource": {
+           "attributes": [
+             {
+               "key": "service.name",
+               "value": { "stringValue": "TestSvcName" }
+             },
+             {
+               "key": "k8s.pod.name",
+               "value": { "stringValue": "second" }
+             }
+           ]
+         },
+         "scopeMetrics": [
+           {
+             "scope": {
+               "name": "spanmetricsconnector"
+             },
+             "metrics": [
+               {
+                 "name": "calls",
+                 "sum": {
+                   "dataPoints": [
+                     {
+                       "attributes": [
+                         {
+                           "key": "service.name",
+                           "value": { "stringValue": "TestSvcName" }
+                         },
+                         {
+                           "key": "span.name",
+                           "value": { "stringValue": "TestSpan" }
+                         },
+                         {
+                           "key": "span.kind",
+                           "value": { "stringValue": "SPAN_KIND_UNSPECIFIED" }
+                         },
+                         {
+                           "key": "status.code",
+                           "value": { "stringValue": "STATUS_CODE_UNSET" }
+                         }
+                       ],
+                       "startTimeUnixNano": "1702582936761872000",
+                       "timeUnixNano": "1702582936761872012",
+                       "asInt": "1"
+                     }
+                   ],
+                   "aggregationTemporality": 2,
+                   "isMonotonic": true
+                 }
+               }
+             ]
+           }
+         ]
+       }
+     ]
+   }
+   ```
+   {{< /collapse >}}
+
```

Comment:
Should we link at the bottom to `Sending metrics via a Prometheus remote write` saying that there's a way to ensure all samples are received?

Reply:
It is not necessary, because using `otelcol.connector.spanmetrics` only with other OTLP-native components is totally ok. There is only a problem when converting the OTLP metrics to Prometheus format.

This "Handling of resource attributes" section simply describes the OTLP features.